gfx-descriptor/src/counts.rs

```diff
@@ -9,9 +9,12 @@ pub use hal::pso::{
     DescriptorType, ImageDescriptorType,
 };
 
-const DESCRIPTOR_TYPES_COUNT: usize = 15;
+/// Length of the [`DESCRIPTOR_TYPES`] array.
```

Comment:
I was thinking about making this `doc(hidden)`, but I guess comments are fine too